app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+# User Controller
+class UsersController < ApplicationController
```

Comment:
Since we are redirecting a user to the page where she can see all her gems, this should probably be called `GemsController`, the path name should match that.